src/Microsoft.ComponentDetection.Detectors/poetry/Contracts/PoetryPackage.cs

```diff
@@ -15,5 +16,9 @@ public class PoetryPackage
 
         [SuppressMessage("StyleCop.CSharp.NamingRules", "SA1300:ElementMustBeginWithUpperCaseLetter", Justification = "Deserialization contract. Casing cannot be overwritten.")]
         public PoetrySource source { get; set; }
+
+        public TomlTable dependencies { get; set; }
```

Comment:
Had to add this property because tomlyn wouldn't create the model if the property didn't exist and their "TomlModelOptions.IgnoreMissingProperties" didnt work in this scenario